src/cmd/manage_nsfs.js

```diff
@@ -142,7 +142,8 @@ async function fetch_bucket_data(action, user_input) {
         path: user_input.path,
         should_create_underlying_storage: action === ACTIONS.ADD ? false : undefined,
         new_name: _.isUndefined(user_input.new_name) ? undefined : String(user_input.new_name),
-        fs_backend: _.isUndefined(user_input.fs_backend) ? config.NSFS_NC_STORAGE_BACKEND : String(user_input.fs_backend)
+        fs_backend: _.isUndefined(user_input.fs_backend) ? config.NSFS_NC_STORAGE_BACKEND : String(user_input.fs_backend),
+        force_md5_etag: _.isUndefined(user_input.force_md5_etag) ? undefined : get_boolean_or_string_value(user_input.force_md5_etag)
```

Comment:
Question - what about a default value (`config.NSFS_CALCULATE_MD5`)?
like we have in `fs_backend` - I saw in the issue a reference to:
https://github.com/noobaa/noobaa-core/blob/d1bf8c465af745dedbff9d3803ec13da344884ba/src/sdk/namespace_fs.js#L1607

Reply:
@naveenpaul1, you're right. When I wrote this comment I thought that maybe we need to use a default value in this property, but we use it in the file `namespace_fs.js` (hence I decided to resolve it).

Reply:
@nadavMiz, the open question I have regarding this line:
https://github.com/noobaa/noobaa-core/blob/d1bf8c465af745dedbff9d3803ec13da344884ba/src/sdk/namespace_fs.js#L1607-L1608

`md5_enabled` will be `true` in one of the cases of the condition to `true`:
1. It was set in the config file (`config.NSFS_CALCULATE_MD5`).
2. It was set in the namespace_fs level (`this.force_md5_etag`).
3. It was set in the account (`object_sdk?.requesting_account?.force_md5_etag`).

The missing case is when it was set in the bucket (as you added in this PR).


Another thing that I don't understand is that if someone sets it in the config, and also adds for a specific account `--force_md5_etag false`, with this logical OR we would ignore it, is that what we want?

Reply:
this logic was added by @achouhan09, so maybe he can way in. but the idea is that `this.force_md5_etag` is the case where it was set in the bucket. so you have three cases:

1. It was set in the config file (config.NSFS_CALCULATE_MD5).
2. It was set in the account (object_sdk?.requesting_account?.force_md5_etag).
3. It was set in the bucket level (this.force_md5_etag).

regarding your other question the way I see it `force_md5_etag` is defaulted to be false unless it is said otherwise anywhere else, either for everybody in the config file, or for a specific bucket / account. 
@achouhan09 is this correct?

Reply:
this is the PR for adding force_md5_etag for buckets: https://github.com/noobaa/noobaa-core/pull/7236
note that this PR only exposes this feature to nsfs cli, it doesn't add new functionality

Reply:
yes @shirady , as @nadavMiz described in above comment that was the logic before. You can also refer to my PRs #7236 & #7297  for more info.

Reply:
@achouhan09 thank you, I just still don't understand this scenario: if in the config file, it was set to `true` but in a specific account it is `false` (not `undefined` the user explicitly set a `false` value) - why do we still `md5_enabled` to be `true`?

Reply:
@shirady here, I think we are only focusing if any of the above 3 cases is true then we need to calculate the MD5 etag, see Danny's comment [here](https://github.com/noobaa/noobaa-core/pull/7236#discussion_r1194156967). He mentioned that we need `config.NSFS_CALCULATE_MD5` as a global toggle, it will allow us to calculate MD5 etag globally.

Reply:
@achouhan09 thank you.
@nadavMiz I will link this discussion in the issue as my open question.
resolving.